rerun_cpp/src/rerun/datatypes/u_int32.cpp

```diff
@@ -0,0 +1,47 @@
+// DO NOT EDIT! This file was auto-generated by crates/re_types_builder/src/codegen/cpp/mod.rs
```

Comment:
weird file-name `u_int32.cpp`

accidentally comited old file?
why wasn't this pruned by `remove_orphaned_files`?